src/schema/v2/artworksForUser/helpers.ts

```diff
@@ -114,19 +118,30 @@ export const getBackfillArtworks = async (
   remainingSize: number,
   includeBackfill: boolean,
   context: ResolverContext,
-  onlyAtAuction = false
+  onlyAtAuction = false,
+  excludeDislikedArtworks = false
 ): Promise<any[]> => {
   if (!includeBackfill || remainingSize < 1) return []
 
   const {
     setItemsLoader,
     setsLoader,
-    authenticatedLoaders: { filterArtworksLoader },
+    authenticatedLoaders: {
+      filterArtworksLoader: filterArtworksAuthenticatedLoader,
+    },
+    unauthenticatedLoaders: {
+      filterArtworksLoader: filterArtworksUnauthenticatedLoader,
+    },
   } = context
 
+  const filterArtworksLoader =
+    excludeDislikedArtworks === true
+      ? filterArtworksAuthenticatedLoader
+      : filterArtworksUnauthenticatedLoader
```

Comment:
We weren't sure if this was the most accurate way to perform this check or not, but the argument isn't specified by any clients yet, so it seems safe to release.